atmo/jobs/urls.py

```diff
@@ -14,4 +14,5 @@
     url(r'^(?P<id>\d+)/edit/', views.edit_spark_job, name='jobs-edit'),
     url(r'^(?P<id>\d+)/run/', views.run_spark_job, name='jobs-run'),
     url(r'^(?P<id>\d+)/$', views.detail_spark_job, name='jobs-detail'),
```

Comment:
The previous view applies for both notebook types. Zeppelin needs that extra end point because we need a page to render Zeppelin notes whereas a Jupyter note will link to an external renderer.